src/lib/auth/update-user-photo.js

```diff
@@ -0,0 +1,36 @@
+import query from '../../db/query';
+import { updateUserPhoto as queryText } from '../../db/sql-queries.json';
+
+/**
+ * updateUserPhoto updates user photo url in the database
+ * @param {object} client - database client
+ * @param {string} user_id - user_id
+ * @param {string} filename - new photo_url
+ * @returns {Promise.<void, Error>}
```

Comment:
I've noticed we've been copying/pasting this line wrong in a lot of our code.
In this case,`{Promise.<void, error>}` should be `{Promise.<string, error>}` if this function returns a string (does it?).

Keep an eye out of other times we've done this, I'm trying to correct them as and when I see it